c++/src/kj/compat/http-test.c++

```diff
@@ -2622,6 +2622,34 @@ KJ_TEST("HttpServer cancels request when client disconnects") {
   cancelPromise.wait(waitScope);
 }
 
+KJ_TEST("HttpServer responds with 400 Bad Request") {
```

Comment:
No tests that actually set a custom error handler?

Reply:
Will write one.